profiles/serializer/normalizer.py

```diff
@@ -0,0 +1,60 @@
+from functools import singledispatch
+from typing import Any
+
+from profiles.models import Affiliation, EmailAddress, Name, Profile
+
+
+@singledispatch
+def normalize(value: Any) -> str:
+    return str(value)
+
+
+@singledispatch
+def normalize_snippet(value: Any) -> str:
+    return str(value)
+
+
+@normalize.register(Profile)
+def normalize_profile(profile: Profile) -> dict:
+    data = normalize_profile_snippet(profile)
+    data['emailAddresses'] = [normalize_snippet(email) for email in profile.email_addresses]
+    data['affiliations'] = [normalize_snippet(aff) for aff in profile.get_affiliations()]
+
+    return data
+
+
+@normalize_snippet.register(Affiliation)
+def normalize_affiliation_snippet(affiliation: Affiliation) -> dict:
```

Comment:
Snippets only exist in the top level (full Profile vs Profile snippet), so I'd use `normalize()`.

Reply:
Same for email addresses.

Reply:
Ok these have been moved to normalise now.